relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -186,11 +187,17 @@ pub(crate) fn extract_span_metrics(
                 );
 
                 if let Some(scrubbed_desc) = sanitized_description {
-                    span_tags.insert(SpanTagKey::Description, scrubbed_desc.to_owned());
+                    // Truncating the span description's tag value is, for now,
+                    // a temporary solution to not get large descriptions dropped. The
+                    // group tag mustn't be affected by this, and still be
+                    // computed from the full, untruncated description.
```

Comment:
I don't really like adding a product requirement in a comment like this, so I'm open to removing it. This case is also covered in the integration test below, but I wouldn't rely too much on that because it depends a lot on the string to find MD5 hashing conflicts.